resources/assets/coffee/react/beatmaps/search-filter.coffee

```diff
@@ -23,8 +23,15 @@ el = React.createElement
 class @SearchFilter extends React.Component
   constructor: (props) ->
     super props
+
+    if @props.default != undefined
+      if @props.multiselect
+        selected = @props.default.split('-')
+      else
+        selected = @props.default
+
     @state =
-      selected: [].concat(@props.default)
+      selected: [].concat(selected)
```

Comment:
undefined/null get filtered out later, but will be redone with the history/popstate rewrite